python/pyspark/ml/tests.py

```diff
@@ -263,6 +263,23 @@ def test_ngram(self):
         transformedDF = ngram0.transform(dataset)
         self.assertEquals(transformedDF.head().output, ["a b c d", "b c d e"])
 
+    def test_stopwordsremover(self):
+        sqlContext = SQLContext(self.sc)
+        data = ["a", "panda"]
+        dataset = sqlContext.createDataFrame([
+            ([data])], ["input"])
```

Comment:
It is just hard to tell what are the rows and what are the fields in this form, e.g., `[(([["a", "b", "c", "d", "e"]])]` vs `[Row(input=["a", "b", "c", "d", "e"])]`. It would be nice if you can update both.